gemd/units/impl.py

```diff
@@ -244,38 +301,23 @@ def convert_units(value: float, starting_unit: str, final_unit: str) -> float:
 
 
 @register_unit_format("clean")
+@deprecated(deprecated_in="2.1.0", removed_in="3.0.0", details="Scaling factor clean-up ")
```

Comment:
if this is deprecated, what is it replaced with?

Reply:
is it that `_unmangle_scaling` encompasses the customization that clean was performing before?

Reply:
Yes, the clean formatter is now obsolete, because of the `__format__` method does its fundamental work without getting in the way of invoking other formatters (`H`, `P`, etc.).  That's potentially very valuable because we'll because to do server-side HTML formatting, rather than forcing a browser side fix.